livekit-plugins/livekit-plugins-elevenlabs/setup.py

```diff
@@ -51,7 +51,7 @@
     python_requires=">=3.9.0",
     install_requires=[
         "livekit >= 0.9.2",
-        "livekit-agents >= 0.3.0",
+        "livekit-agents >= 0.5, < 0.6",
```

Comment:
We don’t want to put a restriction on the maximum used version

Reply:
Use ~=

Reply:
I think we do, we've had issues in the past where we've made a breaking API change (for example to tts) where there is a maximum version of agents that is supported.

For example:
- we change agents tts interface, bump agents to 0.6
- user is on this version of elevenlabs (maybe pinned in their requirements.txt) but maybe they didn't pin agents
- user builds new Docker image. Now they have 11labs at a version that isn't compatible with their version of agents.

Maybe it's an edge case but I think it happens. Especially in local environments where users aren't installing fresh every time.

Reply:
Yes but it is because we break the API and did a patch instead of a major version

Reply:
No i mean if the user pins their requirements to elevenlabs and elevenlabs has no max. They can pull a version of agents that isn't compatible with elevenlabs

Reply:
It is ok if you use ~= 0.6

Reply:
you mean ~= 0.5? yeah I think that is ok. Although we've needed to force an upgrade a few times in the history of agents with patches which is why I have >= but I think ~ will be ok. I'll change it